docs/en/dataset_prepare.md

```diff
@@ -376,3 +393,159 @@ python tools/convert_datasets/isaid.py /path/to/iSAID
+
+The extracted and upsampled COCO objects images and masks can be found in this [drive](https://drive.google.com/drive/folders/15nZETWlGMdcKY6aHbchRsWkUI42KTNs5?usp=sharing).
+
+Please extract CelebAMask-HQ and 11k Hands images based on the splits found in [drive](https://drive.google.com/drive/folders/15nZETWlGMdcKY6aHbchRsWkUI42KTNs5?usp=sharing).
+
+download file to ./data_materials
+
+```none
+CelebAMask-HQ.zip
+CelebAMask-HQ-masks_corrected.7z
+RealOcc.7z
+RealOcc-Wild.7z
+11k-hands_mask.7z
+11k-hands_image.7z
+coco_object.7z
+dtd-r1.0.1.tar.gz
+```
+
+______________________________________________________________________
+
+```bash
+apt-get install p7zip-full
+
+cd data_materials
+
+#extract celebAMask-HQ and split by train-set
+unzip CelebAMask-HQ.zip
+7za x CelebAMask-HQ-masks_corrected.7z -o./CelebAMask-HQ
+#suggest better code if you have
+rsync -a ./CelebAMask-HQ/CelebA-HQ-img/ --files-from=./CelebAMask-HQ-WO-train.txt ./CelebAMask-HQ-WO-Train_img
+basename -s .jpg ./CelebAMask-HQ-train/* > train.txt
+xargs -n 1 -i echo {}.png < train.txt > mask_train.txt
+rsync -a ./CelebAMask-HQ/CelebAMask-HQ-masks_corrected/ --files-from=./mask_train.txt ./CelebAMask-HQ-WO-Train_mask
+mv train.txt ../data/occlusion-aware-face-dataset
+
+#extract DTD
+tar -zxvf dtd-r1.0.1.tar.gz
+mv dtd DTD
+
+#extract hands dataset and split by 200 samples
+7za x 11k-hands_masks.7z -o.
+unzip Hands.zip
+rsync -a ./Hands/ --files-from=./11k_hands_sample.txt ./11k-hands_img
+
+#extract upscaled coco object
+7za x coco_object.7z -o.
+mv coco_object/* .
+
+#extract validation set
+7za x RealOcc.7z -o../data/occlusion-aware-face-dataset
+
+```
+
+**Dataset Organization:**
+
+```none
+
+├── data_materials
+│   ├── CelebAMask-HQ-WO-Train_img
+│   │   ├── {image}.jpg
+│   ├── CelebAMask-HQ-WO-Train_mask
+│   │   ├── {mask}.png
+│   ├── DTD
+│   │   ├── images
+│   │   │   ├── {classA}
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── {classB}
+│   │   │   │   ├── {image}.jpg
+│   ├── 11k-hands_img
+│   │   ├── {image}.jpg
+│   ├── 11k-hands_mask
+│   │   ├── {mask}.png
+│   ├── object_image_sr
+│   │   ├── {image}.jpg
+│   ├── object_mask_x4
+│   │   ├── {mask}.png
+
+├── data
+│   ├── occlusion-aware-face-dataset
+│   │   ├── train.txt
+│   │   ├── NatOcc_hand_sot
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   ├── NatOcc_object
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   ├── RandOcc
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   ├── RealOcc
+│   │   │   ├── img
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── mask
+│   │   │   │   ├── {mask}.png
+│   │   │   ├── split
+│   │   │   │   ├── val.txt
```

Comment:
I think this directory structure should be moved to the end, after the generation scripts.

Reply:
done.